### ISSUE TITLE
Quick upgrade of main

### ISSUE THREAD

**Issue by weberax** [bastislack/highline-freestyle#269]
Hey everyone, I heard you are starting to rewrite the app, nice!
I'm happy to help out, later this winter... Right now I  have only a couple days, in front of an computer.

I guess this rewrite will take some time and thought it would be sad to have all our recent developments (since February) only staying on the develop branch.

So I suggest to make an quick upgrade.

I fixed #223, improved a bit the error message 32c955f91ecca7da1143733d83ab996e06104760 and updated the tricklist #268.
We should probably also merge #264 
Are there any other important bugs/things on the develop branch we need to address?

I could fix some stuff until the 29th, or we upgrade like it is now?

@bastislack @WaldemarLehner @JulianDietzel

**Comment by bastislack** [comment]
Sounds great!
Only one thing: I think we already started with the spanish version right (I mean on the develop branch)? We should make sure this is only on the spanish version pull request so we don't have a 10% working spanish version :D

**Comment by weberax** [comment]
> Sounds great! Only one thing: I think we already started with the spanish version right (I mean on the develop branch)? We should make sure this is only on the spanish version pull request so we don't have a 10% working spanish version :D

I just checked and we only Spanish stuff which is in the develop branch is the implementation of  lingui.js with 3-4 translations. But the language selector was temporary disabled (a0ef883), so looks like no Spanish is accessible/visibly anywhere. So for easiness I would keep this as it is.

**Comment by bastislack** [comment]
Perfect!Am 28.08.23, 13:21 schrieb Axel ***@***.***>:

   
   
   Sounds great! Only one thing: I think we already started with the spanish version right (I mean on the develop branch)? We should make sure this is only on the spanish version pull request so we don't have a 10% working spanish version :D 
   
  I just checked and we only Spanish stuff which is in the develop branch is the implementation of lingui.js with 3-4 translations. But the language selector was temporary disabled (a0ef883), so looks like no Spanish is accessible/visibly anywhere. So for easiness I would keep this as it is.
